specification/containerregistry/resource-manager/readme.python.md

```diff
@@ -20,6 +20,7 @@ Generate all API versions currently shipped for this package
 multiapi: true
 default-api-version: "2025-04-01"
 batch:
+  - tag: package-2025-05-preview-only
   - tag: package-2025-04-only
   - tag: package-2025-03-preview-only
```

Comment:
Please replace old `package-2025-03-preview-only` with new `package-2025-05-preview-only` instead of adding it directly since we don't hope to add total number of tags for python package.